### PR TITLE
Implement basic Journal design

### LINKED ISSUE #61: Daily Journal - Implement Design Prototype
**Is your feature request related to a problem? Please describe.**
As a frontend developer, I want a barebones implementation of the design for the Daily Journal so that I can begin implementing the functionality.

**Describe the solution you'd like**
A react frontend implementation of the prototype design for the Daily Journal. No functionality is required beyond what is required to fully implement the prototype design.

**Describe alternatives you've considered**
The design could be implemented at the same time as the functionality, but this would increase the size of the resulting PR and would be more difficult to test.

**Additional context**
Blocked by #48

### PR DESCRIPTION
**Pull request by lhuijser**
Closes #61 - Daily Journal - Implement Design Prototype

## Proposed Changes

- Added journal png for the background
- Created Journal and Page components
- Added page navigation icons
- Added date texts for each page
- Placed a text area in each page as a placeholder

Screen Shot 2021-03-18 at 10 42 04 AM

### CODE REVIEW

File: telos-frontend/src/App.js [hunk 1]
```diff
@@ -1,23 +1,12 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <div className="Journal-view">
+        <Journal/>
```
Comment (RawiriHohepa) [inline telos-frontend/src/App.js]: Make sure you change the test for learn react link since you've removed it

Comment (RawiriHohepa) [inline telos-frontend/src/App.js]: You should run the linter before you create the pull request

File: telos-frontend/src/views/journal/Journal.jsx [hunk 2]
```diff
@@ -1,3 +1,15 @@
-const Journal = () => <div />;
+import './Journal.css';
+import ArrowForwardIcon from '@material-ui/icons/ArrowForward';
+import ArrowBackIcon from '@material-ui/icons/ArrowBack';
+import Page from './Page';
+
+const Journal = () => (
+<div className="Journal">
+    <ArrowBackIcon className="Arrow-left"/>
```
Comment (RawiriHohepa) [inline telos-frontend/src/views/journal/Journal.jsx]: Good use of Material UI Elements here

**Review by RawiriHohepa — CHANGES_REQUESTED** [review]
Looks mostly good! Just a few small changes to update the tests and run the linter then it's good to approve

File: telos-frontend/src/App.js (hunk 1 above)
Comment (lhuijser) [reply telos-frontend/src/App.js]: Thanks, have changed it now :)

Comment (lhuijser) [reply telos-frontend/src/App.js]: Thank you! Have run it now

**Review by RawiriHohepa — APPROVED** [review]
Looks good now thanks!

**Review by a-158w — APPROVED** [review]
LGTM

**Pull request merged**